src/main/java/forestry/apiculture/ApiaristAI.java

```diff
@@ -0,0 +1,152 @@
+package forestry.apiculture;
+
+import forestry.api.apiculture.EnumBeeType;
+import forestry.apiculture.blocks.BlockApiculture;
+import forestry.apiculture.items.ItemBeeGE;
+import forestry.apiculture.tiles.TileBeeHouse;
+import forestry.core.tiles.TileUtil;
+import forestry.core.utils.InventoryUtil;
+import net.minecraft.block.Block;
+import net.minecraft.entity.ai.EntityAIMoveToBlock;
+import net.minecraft.entity.passive.EntityVillager;
+import net.minecraft.inventory.InventoryBasic;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+@SuppressWarnings("static-access")
+public class ApiaristAI extends EntityAIMoveToBlock{
```

Comment:
Please fix the formatting for spacing around braces and parenthesis in this file.
There are formatting settings for the project here: https://github.com/ForestryMC/ForestryMC/blob/mc-1.12/formatting-settings.jar